.github/workflows/services-all-features.yml

```diff
@@ -4,7 +4,9 @@ on:
   workflow_dispatch:
 
 env:
-  RUSTFLAGS: -Dwarnings
```

Comment:
Is there any way we can keep `-Dwarnings` for the hand rolled sdks for now?

Reply:
This workflow is for building the services with --all-features. It is not for the hand rolled sdks.